tests/default/indices/refresh.yaml

```diff
@@ -37,3 +37,27 @@ chapters:
       payload:
         _shards:
           failed: 0
+  - synopsis: Adding one more document.
```

Comment:
We're not trying to test the server, but the API schema. I don't think this adds anything as we already call `_refresh` above with the same results.